website/cue/reference/remap/functions/uuid_from_friendly_id.cue

```diff
@@ -0,0 +1,34 @@
+package metadata
+
+remap: functions: uuid_from_friendly_id: {
+	category:    "Random"
+	description: """
+		Convert a Friendly ID (base62 encoding a 128-bit word) to a UUID.
+		"""
+
+	arguments: [
+		{
+			name:        "value"
+			description: "A string that is a Friendly ID"
+			required:    true
+			type: ["timestamp"]
+		},
+	]
+	internal_failure_reasons: [
+		"`value` is not a string.",
```

Comment:
```suggestion
```

This is actually validated by the compiler and so it will catch it before runtime.